added IDC collections statistics query, named  query_stats_agg_idc.txt in the root folder.

ccosmin97 opened the pull request:
@fedorov

ccosmin97 committed:
added IDC collections statistics query

ccosmin97 committed:
Update query_stats_agg_idc.txt

corrected aggregation level, from SOPUID level to distinct SerieUID level.

ccosmin97 committed:
Update query_stats_agg_idc.txt

added MagneticFieldStrength and removed comments.

fedorov commented on query_stats_agg_idc.txt:
```diff
@@ -0,0 +1,48 @@
+  MagneticFieldStrength,
+  PixelSpacing[SAFE_OFFSET(0)] as PixelSpacing,
+  CASE
+    WHEN SeriesDescription LIKE '%t2_tse_tra%' AND collection_id = "prostatex" AND Modality = "MR" THEN "t2w_prostateX" 
```
I would suggest to have a column that just defines the contrast of the series (e.g., "T2W"), and not mix contrast type with the collection ID. We already have collection ID in an existing column.

ccosmin97 replied:
ok, i will separate collection_id and SeriesDescription, then group by colllection_id, SeriesDescription.

fedorov commented:
Just to motivate you to improve your git skills:
1. Rebase the commits into a single commit
2. While rebasing, update the commit message to follow https://slicer.readthedocs.io/en/latest/developer_guide/contributing.html
3. Force-push the local content to your branch

@vkt1414 can answer questions! ;-)

ccosmin97 commented:
> Just to motivate you to improve your git skills:
> 
> 1. Rebase the commits into a single commit
> 2. While rebasing, update the commit message to follow https://slicer.readthedocs.io/en/latest/developer_guide/contributing.html
> 3. Force-push the local content to your branch
> 
> @vkt1414 can answer questions! ;-)

git rebase -i HEAD~3

fedorov commented on query_stats_agg_idc.txt:
```diff
@@ -0,0 +1,48 @@
+WITH temp_query as (
```
You can add a comment header explaining what this query does, and even explain individual lines that are complex. Think about yourself when you started working with SQL - would this query be straightforward to understand?

fedorov reviewed (changes requested):
See earlier comments.

fedorov commented:
> Rebase the commits into a single commit

@ccosmin97 please read this: https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line

ccosmin97 committed:
redo structures of repository

ccosmin97 committed:
Delete .DS_Store

ccosmin97 committed:
added analysis_statistical_testing.ipynb

ccosmin97 committed:
Added plots notebook

ccosmin97 committed:
Delete data/.DS_Store

ccosmin97 committed:
redo structures of repository

ccosmin97 committed:
added results tables

ccosmin97 committed:
Add files via upload

added dashboard version of analysis results table, with image_ohif_url and image_PixelSpacingX DICOM-based attributes.

ccosmin97 committed:
added demo folders

ccosmin97 committed:
added metadata dcmqi json folders

ccosmin97 committed:
Add files via upload

IDC query results for prostate_mri_us_biopsy for images/annotations.

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
added prostate158 process on prostate_mri_us_biopsy

ccosmin97 committed:
Created with Colab

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
prostate158 on qin

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
removed prostatex manifests

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
added demo prostatex zonal prostate158

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Delete DEMO_PROSTATE158_nnunet_prostate_mri_us_biopsy.ipynb

ccosmin97 committed:
added prostate158 prostatex prostate notebook

ccosmin97 committed:
renamed notebook

ccosmin97 committed:
Delete demo_notebooks/DEMO_PROSTATE158_prostate_mri_us_biopsy.ipynb

ccosmin97 committed:
added notebook prostate158 prostate_mri_us_biopsy

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
colab

ccosmin97 committed:
colab

ccosmin97 committed:
Colab

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Delete data/idc_manifests/idc_expert_seg_multi_modal_prostatex_zonal.csv

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Colab

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Delete data/idc_manifests/idc_images_multi_modal_prostatex_prostate.csv

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Delete data/idc_manifests/idc_images_multi_modal_prostatex_prostate.csv

ccosmin97 committed:
Update README.md

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
Update README.md

ccosmin97 committed:
Update README.md

ccosmin97 committed:
Update README.md

ccosmin97 committed:
Colab

ccosmin97 committed:
Add files via upload

ccosmin97 committed:
colab

ccosmin97 committed:
Colab

ccosmin97 committed:
Colab

ccosmin97 committed:
Colab

ccosmin97 committed:
Colab

ccosmin97 committed:
Colab

ccosmin97 committed:
Colab
